wemake_python_styleguide/violations/consistency.py

```diff
@@ -1866,3 +1868,37 @@ class AugmentedAssignPatternViolation(ASTViolation):
 
     error_template = 'Found usable augmented assign pattern'
     code = 350
+
+
+@final
+class UnnecessaryLiteralsViolation(ASTViolation):
+    """
+    Forbids the use of unnecessary literals in your code.
+
+    Reasoning:
```

Comment:
The reasoning is technically correct, but is not complete.
I guess that it should be formulated like so: `We discourage using primitive calls to get default type values. There are better ways to get these values.`

We are also missing a solution here.
The solution should be: `Use direct default values of the given type`